askama_shared/src/generator.rs

```diff
@@ -298,6 +301,28 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
         buf.writeln("}");
     }
 
+    fn impl_tide_integrations(&mut self, buf: &mut Buffer) {
+        let ext = self
+            .input
+            .path
+            .extension()
+            .and_then(|s| s.to_str())
+            .unwrap_or("txt");
+
+        self.write_header(buf, "std::convert::TryInto<::askama_tide::Body>", None);
+        buf.writeln(
+            "type Error = ::askama::Error;\n\
```

Comment:
This should probably import `Error` and `Result` from `askama_tide` rather than `askama` directly (I know this is potentially inconsistent with what's currently there for other integrations, but I think it's the right direction for the future).

Reply:
makes sense to me! resolved in f0af044bc0cc564014a46e5b497d6a86a7507afa, and switched to re-exporting all of tide and askama instead of cherry-picking exports